sections/introduction/main.tex

```diff
@@ -15,54 +17,82 @@ \section{Introduction}
 Furthermore, this is also a huge issue for the EMS.
 By blocking the ambulances at the hospital's parking space, valuable time is 
-wasted where ambulances could be attending new patients \cite{eastanglia}.
+wasted where ambulances could be attending new patients~\cite{eastanglia}.
 
 A number of papers have been published that touch upon the use of 
 queueing models together with game theoretic concepts.
-In \cite{FirmCompetition} the authors study a simultaneous price competition 
-between two firms and calculate the Nash equilibrium both for identical and 
-heterogeneous firms. 
-The authors have also extended their model in \cite{FirmCompetition2} by 
+In~\cite{FirmCompetition} the authors study a simultaneous price competition 
+between two firms that are modelled as two distinct queueing systems with a 
+fixed capacity and a combined arrival rate.
+They calculate the Nash equilibrium both for identical and heterogeneous firms
+and show that for the former a pure NE always exist and for the latter a unique
+equilibrium exists where only one firm operates.
+The authors have also extended their model in~\cite{FirmCompetition2} by 
 allowing the players (firms) to choose capacities. 
 A main result from this paper was that equilibria, where both firms operate
 together, are not socially optimal while when both firms operate as a monopoly 
 they are.
-Another extension of \cite{FirmCompetition} was introduced in 
-\cite{FirmCompetitionExtension} where a long-run version of the competition was 
-considered that also had capacity as a decision variable.
-In \cite{knight2017measuring} a normal form game is built that is informed by a 
+Another extension of~\cite{FirmCompetition} was introduced 
+in~\cite{FirmCompetitionExtension} where a long-run version of the competition 
+was considered that also had capacity as a decision variable.
+In~\cite{knight2017measuring} a normal form game is built that is informed by a 
 two-dimensional Markov chain in order to model interactions between critical
 care units.
-An additional paper that focuses on competition is \cite{fan2009short} where
-the authors created a competition between 2 sellers with products having 
-different implementation costs.
-In \cite{sadat2015can} a healthcare application was studied where patients 
+An additional paper that focuses on competition is~\cite{fan2009short} where
+the authors created a competition between two sellers where seller 1 supplies 
+a product instantly and seller 2 is modelled as a make-to-order M/M/1 queue.
+The game that is played requires the two sellers to make a choice on the price 
+of the product and then seller 2 to set a capacity that guarantees a maximum 
+expected delay.
+In~\cite{sadat2015can} a healthcare application was studied where patients 
 could choose between two hospitals, where a utility function is derived that is
 based on patients' perceived quality of life.
+In~\cite{knight2013selfish} the impact that individual behaviour has on the 
```

Comment:
`How individual behaviour based on congestion impacts the efficiency of public services is investigated in~\cite{knight2013selfish}.`